x-pack/plugins/security/public/ui_api/index.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ReactElement } from 'react';
+
+import type { CoreStart } from 'src/core/public';
+
+import type { ChangePasswordProps, PersonalInfoProps } from '../account_management';
+import { getComponents } from './components';
+
+interface GetUiApiOptions {
+  core: CoreStart;
+}
+
+type LazyComponentFn<T> = (props: T) => ReactElement;
+
+export interface UiApi {
+  components: {
+    getPersonalInfo: LazyComponentFn<PersonalInfoProps>;
+    getChangePassword: LazyComponentFn<Pick<ChangePasswordProps, 'user'>>;
```

Comment:
__nit:__ (once you split this one in two interfaces)
```suggestion
    getChangePassword: LazyComponentFn<ChangePasswordProps>;
```

Reply:
Updated here: https://github.com/elastic/kibana/pull/99791/commits/24258211021c0b995ff548521a172acaac11c636